### PR TITLE
Update dependencies and test

### PR DESCRIPTION
**Pull request by smorimoto**
Vulnerabilities in child dependencies of this package cause GitHub security scans to complain. To fix that, this PR makes a massive elimination of whole dependencies.

| branch                    | Prod | Dev |
| ------------------------- | ---- | --- |
| `master`                  | 15   | 532 |
| `smorimoto:npm-audit-fix` | 7    | 233 |

### CODE REVIEW

File: bin/local-ssl-proxy [hunk 1]
```diff
@@ -1,36 +1,46 @@
 #!/usr/bin/env node
 
-var fs = require("fs");
+var fs = require("node:fs");
+var kleur = require("kleur");
 var proxy = require("http-proxy");
```
Comment (smorimoto) [inline bin/local-ssl-proxy:5]: Reduce dependencies by replacing `chalk` with `kleur`.

File: test/commandline.test.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { expect, test } from "vitest";
```
Comment (smorimoto) [inline test/commandline.test.js:1]: Speed up testing by replacing `ava` with `vitest`.

**Comment by smorimoto** [comment]
You can see the CI result here: https://github.com/smorimoto/local-ssl-proxy/actions/runs/4137055622/jobs/7151740392

File: test/commandline.js [hunk 3]
```diff
@@ -1,58 +0,0 @@
...
-import fs from 'fs';
-import commandline from '../dist/commandline';
-
-test('cert (default)', t => {
-  const { cert } = commandline.parse([]);
-  t.truthy(cert);
-});
-
-test('cert', t => {
-  const { cert } = commandline.parse(['--cert', '../resources/localhost.cert']);
```
Comment (smorimoto) [inline test/commandline.js:11]: This is probably one of the test bugs.

File: test/commandline.js [hunk 4]
```diff
@@ -1,58 +0,0 @@
...
-  t.truthy(cert);
-});
-
-test('key (default)', t => {
-  const { key } = commandline.parse([]);
-  t.truthy(key);
-});
-
-test('key', t => {
-  const { key } = commandline.parse(['--key', '../resources/localhost.key']);
```
Comment (smorimoto) [inline test/commandline.js:21]: Same here.

File: test/commandline.test.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+import commandline from "../dist/commandline.js";
+
+test("cert (default)", () => {
+  const { cert } = commandline.parse([]);
+  expect(cert).toBeTruthy();
+});
+
+test("cert", () => {
+  const { cert } = commandline.parse(["--cert", "./resources/localhost.cert"]);
```
Comment (smorimoto) [inline test/commandline.test.js:10]: The test bug mentioned above has been fixed.

File: test/commandline.test.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  expect(cert).toBeTruthy();
+});
+
+test("key (default)", () => {
+  const { key } = commandline.parse([]);
+  expect(key).toBeTruthy();
+});
+
+test("key", () => {
+  const { key } = commandline.parse(["--key", "./resources/localhost.key"]);
```
Comment (smorimoto) [inline test/commandline.test.js:20]: Same here.

**Comment by meshuamam** [comment]
🎉

**Comment by smorimoto** [comment]
ping @cameronhunter I'd like to fix this. Do you have time to review this?

```
└─ local-ssl-proxy@npm:1.3.0 (via npm:1.3.0)
   └─ nomnom@npm:1.8.1 (via npm:^1.8.1)
      └─ underscore@npm:1.6.0 (via npm:~1.6.0)
```

https://github.com/advisories/GHSA-cf4h-3jhx-xvhq

**Comment by cameronhunter** [comment]
I've rebuilt the package with TypeScript and updated all the dependencies. `yarn npm audit -R` isn't reporting any suggestions, so I think we're good here.

**Pull request closed without merging**